qlib/Logger.qm

```diff
@@ -862,83 +900,85 @@ public namespace Logger {
         }
 
         private:internal {
-            #! The lock to to ensure proper serialization with concurrent access
-            RWLock m_lock();
+            #! The lock to ensure proper serialization with concurrent access
+            RWLock lock();
 
-            #! an inactive appender won't accept any logging request
-            bool m_active = False;
+            #! An inactive appender won't accept any logging request
+            bool active = False;
 
             #! filter chain
-            list<LoggerFilter> m_filters;
+            list<LoggerFilter> filters;
 
             #! Appender name
-            string m_name;
+            string name;
 
             #! async queue
-            *LoggerAppenderQueue m_queue;
+            *LoggerAppenderQueue queue;
         }
 
         #! Creates the object
         /**
-            @param name the appender name
+            @param n_name the appender name
         */
-        constructor(string name = "") {
-            m_name = name;
+        constructor(string n_name = "") {
+            name = n_name;
         }
 
         destructor() {
-            #close();  # TODO: cannot call as the target object may be already destroyed
+            #close();  # cannot call as the target object may be already destroyed
```

Comment:
Order of destructing objects dependent each other. The is called is the other destructor. This is comment are placed not anybody repeat logical step in future and won't reinvent wheel.

Reply:
ok, so not 'target object' (like file or so), but the child instance that inherits from this class is already destroyed, and because `close` will trigger an event and then `processEventImpl` will be called and it is an abstract method implemented in the child, that is already destroyed - that's the reason why `close` cannot be called here, right?  

can you please update the comment then?